File: src/2020/3/python/3.1.py
with open("src/2020/3/inputs.txt", "r") as f:
    inputs = list(f.read().splitlines())
    width = len(inputs[0])
    perm = ["O" for i, line in enumerate(
        inputs) if inputs[i][(i * 3) % width] == "#"]
    print(len(perm))

File: src/2020/3/js/3.1.js
fs = require("fs");
fs.readFile("src/2020/3/inputs.txt", "utf8", function (err, data) {
  let inputs = data.split("\n");
  console.time();
  const width = inputs[0].length;
  let counter = 0;
  for (let i = 1; i < inputs.length; i++) {
    counter += inputs[i][(i * 3) % width] === "#" ? 1 : 0;
  }
  console.log(counter);
  console.timeEnd();
});
